utils/access.ts

```diff
@@ -0,0 +1,19 @@
+import { Request, Response, NextFunction } from 'express';
+
+interface CustomRequest extends Request {
+  user: {
+    psychologist: string
+  }
+}
```

Comment:
a good practice is to have a specific directory for our types
request.d.ts (d for declarative)